packages/rocketchat-plugin-handler/server/plugin_handler.js

```diff
@@ -0,0 +1,2 @@
+plugin_handler = {};
```

Comment:
Do not use global variables.
There is 2 ways to do that correctly:
1. Use the namespace `RocketChat` -> `RocketChat.PluginHandler`
2. Define the variable as const `const PluginHandler`, export it and impor on the other packages

And try to keep the camel case style